frontend/src/api/chatbot.ts

```diff
@@ -28,11 +47,24 @@ export const fetchChatbotReply = async (userMessage: string): Promise<Message> =
 
     return createBotMessage(data.reply || getChatbotText('noResponseMessage'));
   } catch (error) {
-    console.error('Chatbot API error:', error);
+    console.error('Chatbot API error while sending a message:', error);
     return createBotMessage(getChatbotText('errorMessage'));
   }
 };
 
+/**
+ * Sends a request to the backend to delete the chat session with session id sessionId.
+ *
+ * @param sessionId - The session id of the chat to delete
+ */
+export const deleteChatSession = async (sessionId: string): Promise<void> => {
+  try{
+    await fetch(`${API_BASE_URL}/api/chatbot/sessions/${sessionId}`, { method: 'DELETE' });
+  } catch(error) {
+    console.error('Chatbot API error while deleting a chat session:', error);
+  }
+};
+
 /**
```

Comment:
I observe that the three API calls all follow a similar fetch, JSON, and error pattern. So, this could be abstracted into a generic utility.